requirements.txt

```diff
@@ -20,7 +20,8 @@ numpy==1.21.2
 pandas==1.3.2
 pickle-mixin==1.0.2
 prompt-toolkit==3.0.20
-psycopg2==2.9.1
+#psycopg2==2.9.1
+psycopg2-binary==2.9.1

```

Comment:
Is that expected to be a dev dependency only - https://www.psycopg.org/docs/install.html#psycopg-vs-psycopg-binary ?